### ISSUE TITLE
Accessibility review op 10 Community Componenten van Utrecht als voorbereiding voor de NL Design System Candidate componenten

### ISSUE THREAD

**Issue by rianrietveld** [nl-design-system/utrecht#2660]
Als voorbereiding op het bouwen van 10 Candidate componenten voor het NL Design System, hebben @MarjonBakker en @rianrietveld een review gedaan van de toegankelijkheid van deze 10 componten bij Utrecht en Amsterdam.

We namen hiervoor de componenten zoals gebouwd zijn in de Storybooks en we hebben ze getest tegen de [WCAG-succescriteria](https://nldesignsystem.nl/wcag) en [NL Design System-richtlijnen](https://nldesignsystem.nl/richtlijnen) die voor de betreffende componenten van toepassing zijn.

Dit om te zien hoe we deze componenten kunnen overnemen voor de NL Design System-Candidate componenten die we nu aan het bouwen zijn op: https://github.com/nl-design-system/candidate/tree/main/packages/storybook-test

## Het onderzoek van de betreffende componenten bij Utrecht

Hieronder staan de issues met onze reviews, ik hoop dat ze duidelijk zijn en jullie het in je werk kunnen toepassen.
Vragen of opmerkingen? Stel ze in Slack of bij de betreffende issues en mention Marjon of Rian.

- [Paragraph](https://github.com/nl-design-system/candidate/issues/204)
- [Number Badge](https://github.com/nl-design-system/candidate/issues/203)
- [Mark](https://github.com/nl-design-system/candidate/issues/201)
- [Link](https://github.com/nl-design-system/candidate/issues/199)
- [Data Badge](https://github.com/nl-design-system/candidate/issues/197)
- [Color Sample](https://github.com/nl-design-system/candidate/issues/196)
- [Code Block](https://github.com/nl-design-system/candidate/issues/195)
- [Code](https://github.com/nl-design-system/candidate/issues/193)
- [Skip Link](https://github.com/nl-design-system/candidate/issues/166)
- [Heading](https://github.com/nl-design-system/candidate/issues/164)
